language/tools/move-cli/src/lib.rs

```diff
@@ -70,6 +71,7 @@ pub enum Command {
     New(New),
     Prove(Prove),
     Test(Test),
+    Upload(Upload),
```

Comment:
```suggestion
    MoveyUpload(Upload),
```

Reply:
I think this rename is important to avoid confusion with uploading to the blockchain.